web/app/js/components/ResourceDetail.jsx

```diff
@@ -179,6 +186,14 @@ export class ResourceDetailBase extends React.Component {
             api={this.api} />
         </div>
 
+        <div className="page-section">
+          <TopModule
+            pathPrefix={this.props.pathPrefix}
+            query={topQuery}
+            startTap={true}
+            maxRowsToDisplay={15} />
```

Comment:
Clicking around in the UI, 15 rows feels like a few too many to show for this table. It pushes the upstream/downstream tables way down the page. How would you feel about display 5 or 10 instead? I think the mocks show 5.

Reply:
yeah I agree that 15 feels like too much. I am okay with 5, though for things like the voting service those might not be enough. we can go with 5 for now and see how we feel!